common/app/model/meta.scala

```diff
@@ -295,8 +508,9 @@ trait Elements {
 /**
  * Tags lets you extract meaning from tags on a page.
  */
-trait Tags {
-  def tags: Seq[Tag] = Nil
+final case class Tags(
+  tags: Seq[Tag]) {
```

Comment:
Perhaps this val should be called `all` or something like that, to distinguish it from a particular subset of tags?